olwebsite-app/src/components/Toolbar/__tests__/HamburgerToolbar.test.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 
 import HamburgerToolbar from '../HamburgerToolbar';
-import { renderWithRouter, getComponentFromDOM, clickEventByLabelText} from "test-utils";
+import { renderWithRouter, clickEventByLabelText} from "test-utils";
+
+import { queryByTestId } from '@testing-library/react';
```

Comment:
Don't forget that you can get all the @testing-library exports from test utils, so just lump queryByTestId in that one import!